### PR TITLE
CLDR-15707 Survey Tool hung threads; resurrect cldr-apps-webdriver

### PR DESCRIPTION
**Pull request by btangmu**
-Revise to use Maven (pom.xml) and JUnit

-Further work is needed before tests will pass

### CODE REVIEW

**Comment by btangmu** [comment]
@srl295 this is still very rough -- my goal was to get the old code mostly working again, with new dependencies on Maven and JUnit -- any suggestions for improving pom.xml and/or the overall structure?

File: src/main/java/org/unicode/cldr/surveydriver/App.java [hunk 1]
```diff
@@ -0,0 +1,12 @@
...
+/**
+ * Hello world!
+ *
+ */
+public class App {
+
+  public static void main(String[] args) {
+    System.out.println("Hello World! -- this is main");
+  }
+}
```
Comment (btangmu) [inline src/main/java/org/unicode/cldr/surveydriver/App.java:12]: I suppose we should get rid of App.java, assuming all the real code will be under src/test/java/...

File: src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java [hunk 2]
```diff
@@ -127,9 +128,6 @@ public static void main(String[] args) {
 		if (TEST_ANNOTATION_VOTING) {
 			s.testAnnotationVoting();
 		}
-		if (TEST_XML_UPLOADER) {
-			new SurveyDriverXMLUploader(s).testXMLUploader();
-		}
```
Comment (btangmu) [inline src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java]: I'll restore the deleted parts for TEST_VETTING_TABLE and TEST_XML_UPLOADER; I deleted them while testing a minimal test version without those two files

File: src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java [hunk 3]
```diff
@@ -500,10 +507,10 @@ public boolean login() {
 			return false;
 		}
 		/*
-		 * To make sure we're really logged in, find an element with class "glyphicon-log-out".
+		 * To make sure we're really logged in, find an element with class "glyphicon-user".
 		 */
-		if (!waitUntilClassExists("glyphicon-log-out", true, url)) {
-			System.out.println("❌ Login failed, log-out icon never appeared in " + url);
+		if (!waitUntilClassExists("glyphicon-user", true, url)) {
+			System.out.println("❌ Login failed, glyphicon-user icon never appeared in " + url);
```
Comment (btangmu) [inline src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java:509]: this change is needed due to changes in Survey Tool

File: src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java [hunk 4]
```diff
@@ -611,6 +621,7 @@ private void testAllLocalesAndPages() {
 				}
 			}
 		}
+		*/
```
Comment (btangmu) [inline src/test/java/org/unicode/cldr/surveydriver/SurveyDriver.java]: please excuse the commented-out code here -- I'll follow up with another PR to clean it up

**Review by srl295 — APPROVED** [review]

**Pull request merged**